fiddler/assets/dashboards/fiddler_overview.json

```diff
@@ -0,0 +1 @@
t":3}},{"id":3361228858411406,"definition":{"title":"Model Performance Metrics","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"time":{},"type":"timeseries","requests":[{"formulas":[{"formula":"query1"},{"formula":"query2"},{"formula":"query3"},{"formula":"query4"},{"formula":"query5"},{"formula":"query6"},{"formula":"query7"}],"response_format":"timeseries","queries":[{"query":"avg:fiddler.fpr{$project,$model}","data_source":"metrics","name":"query1"},{"query":"avg:fiddler.f1_score{$project,$model}","data_source":"metrics","name":"query2"},{"query":"avg:fiddler.tpr{$project,$model}","data_source":"metrics","name":"query3"},{"query":"avg:fiddler.accuracy{$project,$model}","data_source":"metrics","name":"query4"},{"query":"avg:fiddler.auc{$project,$model}","data_source":"metrics","name":"query5"},{"query":"avg:fiddler.precision{$project,$model}","data_source":"metrics","name":"query6"},{"query":"avg:fiddler.recall{$project,$model}","data_source":"metrics","name":"query7"}],"style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]},"layout":{"x":6,"y":4,"width":6,"height":3}},{"id":894816583866318,"definition":{"title":"Output Drift","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"time":{},"type":"timeseries","requests":[{"formulas":[{"formula":"query1"}],"queries":[{"query":"avg:fiddler.histogram_drift{feature:probability_churn}","data_source":"metrics","name":"query1"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"},{"formulas":[{"formula":"query0"}],"queries":[{"query":"avg:fiddler.histogram_drift{feature:prob_loan_status}","data_source":"metrics","name":"query0"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]},"layout":{"x":0,"y":7,"width":6,"height":3}},{"id":6241039958772764,"definition":{"title":"Feature Averages Over Time","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"time":{},"type":"timeseries","requests":[{"formulas":[{"formula":"query1"}],"queries":[{"query":"avg:fiddler.output_average{$project,$model,$feature} by {feature}","data_source":"metrics","name":"query1"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]},"layout":{"x":6,"y":7,"width":6,"height":3}}],"template_variables":[{"name":"project","prefix":"project","available_values":[],"default":"*"},{"name":"model","prefix":"model","available_values":[],"default":"*"},{"name":"feature","prefix":"feature","available_values":[],"default":"*"}],"layout_type":"ordered","is_read_only":false,"notify_list":[],"reflow_type":"fixed","id":"zeh-g4g-tkv"}
```

Comment:
CAn you update the title of the dashboard without the dash? Ex: Fiddler Overview

Reply:
@emarsha94 -- are you asking if I can change the title of the asset /images/fiddler-overview?  Or change the image itself to have the screenshot of the dashboard example be "Fiddler Overview"?

Reply:
Never mind, I understand what you're asking....